entgql/internal/todo/resolver.go

```diff
@@ -26,5 +26,8 @@ type Resolver struct{ client *ent.Client }
 func NewSchema(client *ent.Client) graphql.ExecutableSchema {
 	return NewExecutableSchema(Config{
 		Resolvers: &Resolver{client},
+		Directives: DirectiveRoot{
+			HasPermissions: HasPermission(),
+		},
```

Comment:
I think it's preferred to keep our examples simpler. The expected testing from our side is to check the generated schema, the rest if gqlgen logic.

Reply:
I completely understand. I put this here because I can't run the tests without it, cause of the unimplemented directive.
> [{"message":"directive hasPermissions is not implemented","path":["users","edges",0,"node","groups","edges",0,"node"]}]